components/automate-cs-oc-erchef/habitat/config/sys.config

```diff
@@ -92,12 +92,14 @@
 
         {authz_timeout, {{cfg.authz.timeout}}},
         {authz_fanout, {{cfg.authz.fanout}}},
+        {authz_cleanup_batch_size, {{cfg.authz.cleanup_batch_size}}},
```

Comment:
We do not need to add a new config. Add the config to L158. 
Modify 
`{cleanup_batch_size, 0}` to `{cleanup_batch_size, {{cfg.authz.cleanup_batch_size}}}`